activerecord/test/cases/base_test.rb

```diff
@@ -1661,6 +1661,14 @@ def test_protected_environments_are_stored_as_an_array_of_string
     end
   end
 
+  test "cannot call connects_to on non-abstract or non-Base classes" do
```

Comment:
```suggestion
  test "cannot call connects_to on non-abstract or non-ActiveRecord::Base classes" do
```
Just to be consistent.